.github/workflows/mvn-build-and-check-streamx-cli-commands-linux.yaml

```diff
@@ -0,0 +1,58 @@
+name: Check streamX CLI commands for linux
+
+on:
+  workflow_dispatch:
+    inputs:
+      streamx_source:
+        type: choice
+        description: Choose to run test against built streamx or installed from public repos
+        options:
+          - built
+          - installed
+        required: true
+        default: 'built'
+  push:
```

Comment:
I think, we should remove this `push`. For now, this will be only manual action.